code/modules/power/rbmk/_rbmk_defines.dm

```diff
@@ -31,3 +31,6 @@
 #define REACTOR_EMERGENCY_PERCENT 25
 #define REACTOR_DANGER_PERCENT 50
 #define REACTOR_WARNING_PERCENT 100
+
+//Blame Aramix for this monstrosity. In case you are wondering what this is, it is exp(0.005669), which is used for graphing reactor damage calculations (constant)
+#define REACTOR_DAMAGE_MAGIC_NUMBER 1.0056850991882220077025096652119341723277747452195921845851
```

Comment:
as far as I know, Byond can't handle this long shit decimals. 
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/87972842/585034b6-0a28-4053-93e0-ac37cc99730d)